package.json

```diff
@@ -2,7 +2,9 @@
   "name": "react-onclickoutside",
   "version": "5.11.1",
   "description": "An onClickOutside wrapper for React components",
-  "main": "index.js",
+  "main": "./lib/index.js",
```

Comment:
then every user of `webpack2` or `rollup` would have to include in their apps the `umd` bloat (require wrapper) for no good reason, this allows every bundler to chose the best (minimal) build